app/services/open_states_service.rb

```diff
@@ -12,6 +12,23 @@ def initialize
     }
   end
 
+  def enumerate_all(fetch_by_page, query = {})
```

Comment:
this is cool. what do you think about renaming this to `enumerate_pages` to add a bit more specificity? should probably also make it private and update `fetch_bills` to use it as well.

Reply:
I was gonna make it private, but figured I should have a discussion with you first. I was actually deciding between `enumerate_pages`, `enumerate_all` and `enumerate_all_pages` but (1) I wanted to keep to two words and (2) felt that `enumerate_pages` sounded like some potentially non-maximal enumeration. Arbitrary reasoning aside, `enumerate_pages` could probably work too.

Reply:
i think either `enumerate_pages` or `enumerate_all_pages` is cool, all is certainly more explicit.